src/Phinx/Console/Command/Create.php

```diff
@@ -131,7 +194,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
+        // Do we need to do the post creation call to the creation class?
+        if (!empty($creationClassName)) {
+            try {
+                $creationClass->postMigrationCreation($filePath, $className, $this->getConfig()->getMigrationBaseClassName());
+            } catch(\Exception $ex) {
+                throw new \RuntimeException(sprintf(
+                    'Problem calling %s->postMigrationCreation(), resulting in %s',
+                    $creationClassName,
+                    $ex->getMessage()
+                ));
+            }
+        }
+
         $output->writeln('<info>using migration base class</info> ' . $classes['$useClassName']);
+
+        if (!empty($altTemplate)) {
+            $output->writeln('<info>using alternative template</info> ' . realpath($altTemplate));
+        } elseif (!empty($creationClassName)) {
+            $output->writeln('<info>using template creation class</info> ' . $creationClassName);
+        } else {
+            $output->writeln('<info>using template</info> ' . realpath(dirname(__FILE__) . self::DEFAULT_MIGRATION_TEMPLATE));
```

Comment:
The template may be synthetic and not physical and that is why we don't use altTemplate. The CreationInterface IS about the creation of the template for the migration, not just the supplying of a potentially non-existent template filename.